dev/com.ibm.ws.transport.http/test/com/ibm/ws/http/channel/test/api/SameSiteCookieUtilsTest.java

```diff
@@ -0,0 +1,259 @@
+                                                                   //Chrome 60
+                                                                   "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.78 Safari/537.36",
+                                                                   //Chrome 61
+                                                                   "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.79 Safari/537.36",
+                                                                   //Chrome 62
+                                                                   "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3165.0 Safari/537.36",
+                                                                   //Chrome 63
+                                                                   "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3213.3 Safari/537.36",
+                                                                   //Chrome 64
+                                                                   "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.119 Safari/537.36",
+                                                                   //Chrome 65"
+                                                                   "Mozilla/5.0 (Win) AppleWebKit/1000.0 (KHTML, like Gecko) Chrome/65.663 Safari/1000.01",
+                                                                   //Chrome 66
+                                                                   "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3334.0 Safari/537.36",
+                                                                   //Chrome 66 Webview
+                                                                   "Mozilla/5.0 (Linux; Android 4.4.4; One Build/KTU84L.H4) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/66.0.0.0 Mobile Safari/537.36 [FB_IAB/FB4A;FBAV/28.0.0.20.16;]"
+
+            };
+
+            for (String currentUserAgent : incompatibleChromeUserAgents) {
+                assertTrue(SameSiteCookieUtils.isSameSiteNoneIncompatible(currentUserAgent));
+            }
+
+            //Versions Chrome 67 forward support SameSite=None. Test a supported newer version.
+            //Expected to return false.
+            //Chrome 67
+            userAgent = "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.2526.73 Safari/537.36";
+            assertFalse(SameSiteCookieUtils.isSameSiteNoneIncompatible(userAgent));
+
+            //Test UC browser before 12.13.2
+            //Test UC browser on 12.13.2
+            //Test UC browser after 12.13.2
+            //test bad format of product
```

Comment:
This comment looks like it can be removed since the UC tests are next.

However, I don't see a bad format test? Could you create one?